versioned_docs/version-v5/deployment/app-store.md

```diff
@@ -43,17 +43,16 @@ There are two types of certificates that matter here, Development, and Distribut
 
 Distribution certs are meant for distributing an app to the store. When an app is signed with a Distribution cert, it can be installed on any device.
 
-## Signing the App in Xcode.
+## Signing the App in Xcode
```

Comment:
You might need to rebase off feature-7.1. It looks like there are changes in here that aren't part of the input masking work.

Reply:
Yeah I might need to sync `feature-7.1` with `main`.